.vscode/tasks.json

```diff
@@ -1,6 +1,4 @@
 {
-    // See https://go.microsoft.com/fwlink/?LinkId=733558
-    // for the documentation about the tasks.json format
```

Comment:
Hm, this is a bit aggressive from the check-json tool even though officially json does not support comments. Btw I'm not really sure the repo needs a .vscode/tasks.json file at all

Reply:
Yeah, comments are not allowed like this in json. I would probably leave removal this file for another PR as I don't know the context for it.

Reply:
May be we can remove this dir and add it to .gitignore.

Reply:
We discussed that earlier. That can be a separate PR, feel free to propose that.